src/InitOMPasses.hpp

```diff
@@ -38,7 +38,17 @@ void initOMPasses(int optLevel) {
   });
 
   mlir::registerPass([]() -> std::unique_ptr<mlir::Pass> {
-    return createShapeInferencePass();
+    return createONNXHybridTransformPass();
+  });
+
+  mlir::registerPass([]() -> std::unique_ptr<mlir::Pass> {
+    if (enableONNXHybridPass) {
+      // Replaces shape inference in onnx-mlir-opt which is used in all the
+      // lit tests, except the parse lit tests.
+      return createONNXShapeInferenceTransformPass();
```

Comment:
I think that we can add both createONNXHybridTransformPass and createONNXShapeInferenceTransformPass and keep the ShapeInferencePass as it was. We can copy/add new lit test for the new passes without  touch the existing lit test. The ShapeInferencePass will be retired after ONNXShapeInferenceTransformPass becomes stable.